pkg/ccl/backupccl/backupinfo/backup_metadata.go

```diff
@@ -37,6 +37,10 @@ import (
 const (
 	// MetadataSSTName is the name of the SST file containing the backup metadata.
 	MetadataSSTName = "metadata.sst"
+	// SlimManifestFileInfoPath is the name of the SST file containing the
+	// BackupManifest_Files of the backup. This file is always written in
+	// conjunction with the `SLIM_BACKUP_MANIFEST`.
+	SlimManifestFileInfoPath = "slim_manifest_fileinfo.sst"
```

Comment:
bikeshed: donno this needs to mention slim or manifest; maybe just `filelist.sst` ?